CMakeLists.txt

```diff
@@ -202,10 +202,13 @@ endif()
 set(CARGO_TARGET_DIR "${CMAKE_BINARY_DIR}/${BUILD_DIR}/cargo/build")
 
 # Add CMake tests for `cargo test/clippy/fmt/doc`.
-add_test(NAME cargo_tests COMMAND cargo test --features link_qt_object_files --all-targets --target-dir
-                                  ${CARGO_TARGET_DIR})
-add_test(NAME cargo_doc_tests COMMAND cargo test --features link_qt_object_files --doc --target-dir
```

Comment:
can `--all-targets` and `--exclude` not be combined? So that it does "all except qml-meta-project"

Reply:
`--exclude` only works with `--workspace`, not `--all-targets`